PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalClient.kt

```diff
@@ -93,8 +99,20 @@ class PayPalClient internal constructor(
             error: Exception? ->
             if (payPalResponse != null) {
                 payPalContextId = payPalResponse.pairingId
+                linkType = if (internalPayPalClient.isAppSwitchEnabled(payPalRequest) &&
+                    internalPayPalClient.isPayPalInstalled(context)) {
+                    LinkType.UNIVERSAL
+                } else {
+                    LinkType.DEEPLINK
```

Comment:
For PayPal in v5 all links are going to be universal/app links - we no longer support deep links

Reply:
I think this actually indicates if we fall back to the legacy (CCT) flow or are using the app switch flow

Reply:
That's correct, this is just for analytics purposes. In this case, we're setting the `linkType` to differentiate the type of flow that's happening. If the user is eligible for AppSwitch, we send the parameter as `UNIVERSAL`,  otherwise, we send it as `DEEPLINK`

Reply:
Can we internally rename these to `APP_LINK` and `APP_SWITCH`? If we need to send certain values for analytics purposes, we could have a string value for each of these. I think it'd make things a bit clearer inside the SDK.